js/src/apbct-public--7--trp.js

```diff
@@ -70,4 +70,73 @@ document.addEventListener('DOMContentLoaded', function() {
         element.append(trpLayout);
         element.append(trpDescription);
     });
+
+    const badges = document.querySelectorAll('.apbct-real-user-badge');
+
+    badges.forEach((badge) => {
+        let hideTimeout = undefined;
+
+        badge.addEventListener('click', function() {
+            const popup = this.nextElementSibling;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/nextElementSibling

popup can be null, handle this case

Reply:
nice catch, thanks, fixed https://github.com/CleanTalk/wordpress-antispam/pull/501/commits/f26fe6e5d8f22fd7b99a34e3f5b700ed5224f76a